open-metadata-implementation/common-services/ffdc-services/src/main/java/org/odpi/openmetadata/commonservices/ffdc/rest/GenericResponse.java

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.commonservices.ffdc.rest;
+
+import java.util.Collection;
+import java.util.List;
+
+
```

Comment:
Please could you document this interface. I suggest something like. "If an OMAS uses Generic types to implement it's Java API, then the responses can implement this interface."